packages/web/components/ui/media-placeholder-node.tsx

```
@@ -137,29 +141,29 @@
     return (
-      <PlateElement className="my-1" {...props}>
+      <PlateElement className='my-1' {...props}>
         {(!loading || !isImage) && (
           <div
             className={cn(
-              'flex cursor-pointer items-center rounded-sm bg-muted p-3 pr-9 select-none hover:bg-primary/10',
+              'flex cursor-pointer items-center rounded-sm bg-muted p-3 pr-9 select-none hover:bg-primary/10'
             )}
             onClick={() => !loading && openFilePicker()}
             contentEditable={false}
           >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)